var/spack/repos/builtin/packages/py-pyarrow/package.py

```diff
@@ -26,7 +27,7 @@ class PyPyarrow(PythonPackage):
     depends_on('py-setuptools', type='build')
     depends_on('py-cython', type='build')
```

Comment:
The latest version includes:
```python
install_requires = (                                                            
    'numpy >= 1.14',                                                            
    'six >= 1.0.0',                                                             
    'futures; python_version < "3.2"',                                          
    'enum34 >= 1.1.6; python_version < "3.4"',                                  
)  
setup_requires=['setuptools_scm', 'cython >= 0.29']
tests_require=['pytest', 'pandas', 'hypothesis',                            
               'pathlib2; python_version < "3.4"'],    
```
You'll have to see what version added these dependencies.

Reply:
I just restricted this to the versions newer than `0.15.0` we don't have any pypi-originated sources anyways, but this is really a certain amount of guess work... Thanks for your patience